Assets/MixedRealityToolkit/Utilities/Scripts/SceneContentScaler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+using UnityEngine;
+#if UNITY_WSA && UNITY_2017_2_OR_NEWER
+using UnityEngine.XR.WSA;
+#endif
+
+public class SceneContentScaler : MonoBehaviour
+{
+    public enum HMDType
```

Comment:
Do we really need this?  `HolographicSettings.IsDisplayOpaque` should be enough to tell the difference.
Maybe we can have some global value we can set this at, but maybe that's for vNEXT.